source/support/fob.sqf

```diff
@@ -1,14 +1,30 @@
+/*
+    File: fob.sqf
+
+    Author: Kibot
+
+    Description:
+        Searches for a position and setups up Forward Operating Base
+
+    Parameter(s):
+        _this select 0 - OBJECT - A non-nil object with a physical location.
+
+    Usage:
+        _scriptHandle = [player] execVM 'fob.sqf';
+
+    Returns:
+        - Nil -
+*/
+
 _position = _this select 0;
 _size = _this select 1;
```

Comment:
Doesn't match the Header. That says there is only one parameter.
Also use `params`